Add files via upload

sulton1995 opened the pull request:
the task is done. Can you please check it?

sulton1995 committed:
Add files via upload

ScheiklP commented:
Hi @sulton1995 have you checked that the code actually runs?
For example by running the tests in the test folder, or running `python3 sofa_env/scenes/controllable_object_example/controllable_env.py`

sulton1995 committed:
Add files via upload

i did some small changes and it seems like working. But I am facing some problems with installation of sofa. to make the tests you mentioned

ScheiklP commented:
Hi @sulton1995, I am sorry but I cannot review the code like this. 

You do not initialize a pygame window, so trying to read from it in the function will not work.

Also

```python
    def get_rgb(self,render_fr):
        if render_fr == Framework.PYGLET:
            return self.get_rgb_from_pygame()
        elif render_fr == Framework.PYGAME:
            return self.get_rgb_from_pygame()
  ```

calls the same function.

I will close the PR.
Feel free to properly test your implementation and open a new PR at a later time.
